### ISSUE TITLE
"Localplay Control" interface with MPD does not display song/file names in "Current Playlist"

### ISSUE THREAD

**Issue by MartNytrm** [ampache/ampache#858]
Hello,

I'm using Ampache (ver. 3.8.0-beta1) with MPD (ver. 0.18.7) among other things. Everything works fine except the fact, that the "Localplay Control" interface doesn't show any titel/file information in the "Current Playlist" section but "unknown" for every track. Under "Now Playing", the file name of the actually played song is shown, but only after a complete site refresh. It is the same for loading a single song in the Localplay playlist or a whole playlist.
Tags of songs are working correctly for Ampache as well as for MPD alone.
I only found this here: https://github.com/ampache/ampache/issues/158

![selection_008](https://cloud.githubusercontent.com/assets/11982465/7652519/b0243f2a-fb0d-11e4-8632-6dad12592166.png)
![selection_009](https://cloud.githubusercontent.com/assets/11982465/7652520/b02ec170-fb0d-11e4-8233-16c0f4cee821.png)

I'm not sure if I do something wrong. I tried all i could imagine, but without success. Maybe it is a bug, I don't know.
Does anybody have suggestions or ideas what the reason for this issue could be?
Thanks!

Greets

Nytrm

**Comment by Afterster** [comment]
Lets call the ones involved previously :)
@jhermsen? @stebe? Thanks!

**Comment by MartNytrm** [comment]
Ok, I now understand the "issue".

Under Localplay's "Current Playlist" tags of songs are only shown if you open a playlist(-file), which is in the MPD playlist directory, over MPD.

If you open an Ampach playlist (DB) in Ampache for Localplay, you won't see any information, but this playlist will be played, though.

This "issue" can be closed. It's not a real bug, but at least a "bad behavior", because for me it makes the ampache feature "Controlling MPD with Ampache" useless.
What I would appreciate is: Creating playlist in ampache, play this playlists over Localplay/MPD AND see the actual playing playlist so that one can see the songs and find the right one.

But thanks for the help and for this awesome software ampache! Keep on!

Greets


Nytrm

**Comment by michaelcadilhac** [comment]
So why is it that the MPD Localplay playlist cannot be displayed into the local web player?  Indeed, at this point, I'm not even able to play/pause a remote MPD instance, let alone have some fun with the current playlist, since the control are just not available.

My config is that the remote server X has all my music, while my local home server Y has just an MPD bridge that should be fed by X.  The end goal is to have access to my music at work (directly on a browser connected to X) and at home (on a computer on the same network as Y) through X's interface.  This would require to have Localplay to behave, in surface, exactly the same as the web player.  Is it doable/in the boxes?  Cheers!

**Comment by wagnered** [comment]
@michaelcadilhac .

 This issue has been  closed for a long time and isn't related to your apparent problem.  Please create a new issue to ask your question. It will be easier for others to follow the conversation.

Thanks...

**Comment by michaelcadilhac** [comment]
Sorry, I felt that this was more of a side question on the usability of MPD control in Ampache, a topic that was at the heart of that thread.  I'll go on hop on issue #570 which seems more appropriate.  Sorry for the noise.
